libs/signal-store/src/lib/models.ts

```diff
@@ -3,7 +3,7 @@ import { Observable } from 'rxjs';
 import { Action, StateOrCallback } from '@mini-rx/common';
 
 export interface ComponentStoreLike<StateType> {
-    state: Signal<StateType>;
+    get state(): StateType;
```

Comment:
[Writing it like this will not actually enforce the use of a getter](https://stackoverflow.com/a/12839104), the interface will just verify that there is a public property `state` that returns a value of type `StateType`. So imo, better to just type this as `state: StateType` to avoid any confusion.

Reply:
Not sure what is better.

Even if a getter is not enforced with that interface, the intention becomes more clear: use a getter.

For me both ways are good. 
You decide :)

Reply:
In my opinion, when there's a healthy discussion, the author should have the final decision in what route to take. So I'll leave it up to you.